packages/legend-extension-dsl-data-space-studio/src/components/DataSpaceEditor.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+import { useEditorStore } from '@finos/legend-application-studio';
+import { PanelFormTextField, PanelFormSection } from '@finos/legend-art';
+import { DataSpaceEditorState } from '../stores/DataSpaceEditorState.js';
+
+export const DataSpaceEditor = observer(() => {
+  const editorStore = useEditorStore();
+
+  const formEditorState =
+    editorStore.tabManagerState.getCurrentEditorState(DataSpaceEditorState);
+
+  const formElement = formEditorState.dataSpace;
+
+  const handleTitleChange = (value: string | undefined) => {
+    formElement.title = value ?? '';
```

Comment:
these should be wrapper in modifiers, 
see: `DSL_Text_GraphModifierHelper`

Reply:
Fixed